engine/src/test/java/io/camunda/zeebe/engine/state/message/MessageStateTest.java

```diff
@@ -192,7 +194,8 @@ public void shouldNotVisitMessagesBeforeTime() {
 
     // then
     final List<Long> readMessage = new ArrayList<>();
-    messageState.visitMessagesWithDeadlineBefore(1_000, readMessage::add);
+    messageState.visitMessagesWithDeadlineBeforeTimestamp(
```

Comment:
:wrench: might make sense to assert the return value, then we might had catched the issue from above.